js/src/edit-block/components/main.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { PostTitle } from '@wordpress/editor';
+
+/**
+ * The main editing area component.
+ *
+ * Todo: add the rest of this and make it dynamic.
+ *
+ * @return {React.ReactElement} The main editing area.
+ */
+const Main = () => {
+	const urlBlockTemplates = 'https://developer.wpengine.com/genesis-custom-blocks/get-started/add-a-custom-block-to-your-website-content/';
+	const urlTemplateFunctions = 'https://developer.wpengine.com/genesis-custom-blocks/';
+
+	return (
+		<div className="flex flex-col flex-grow items-start w-full overflow-scroll">
+			<div className="flex flex-col w-full max-w-2xl mx-auto pb-64">
+				<div className="block-title-field w-full mt-10 text-center focus:outline-none">
+					<PostTitle />
+				</div>
+				<div className="mt-4 p-5 bg-blue-100 text-blue-700 border-l-4 border-blue-700 rounded-sm">
+					<div className="flex items-center">
+						<svg className="w-6 h-6 fill-current" fill="currentColor" viewBox="0 0 20 20">
+							<path
+								fillRule="evenodd" d="M6.267 3.455a3.066 3.066 0 001.745-.723 3.066 3.066 0 013.976 0 3.066 3.066 0 001.745.723 3.066 3.066 0 012.812 2.812c.051.643.304 1.254.723 1.745a3.066 3.066 0 010 3.976 3.066 3.066 0 00-.723 1.745 3.066 3.066 0 01-2.812 2.812 3.066 3.066 0 00-1.745.723 3.066 3.066 0 01-3.976 0 3.066 3.066 0 00-1.745-.723 3.066 3.066 0 01-2.812-2.812 3.066 3.066 0 00-.723-1.745 3.066 3.066 0 010-3.976 3.066 3.066 0 00.723-1.745 3.066 3.066 0 012.812-2.812zm7.44 5.252a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z"
+								clipRule="evenodd"
+							/>
+						</svg>
+						<h4 className="text-lg font-semibold text-blue-900 ml-2">
+							{ __( 'Next step: Build your block template.', 'genesis-custom-blocks' ) }
+						</h4>
+					</div>
+					<p className="text-sm mt-2 ml-2">
+						{ __( 'To display this block, the plugin will look for this template file in your theme:', 'genesis-custom-blocks' ) }
+					</p>
+					<p className="w-auto text-xs font-mono mt-2 ml-2 px-2 py-1 bg-blue-200 rounded-sm">
+						{ '/wp-content/themes/twentytwenty-child/blocks/' }
```

Comment:
Not necessarily crucial at this stage, but a reminder that it would be nice to display the path of the active theme. Two things I think... 
- `/wp-content/themes/` probably needs to be passed in or retrieved from somewhere since it can be changed via `WP_CONTENT_DIR` and/or related theme functions that are filterable.
- actual active theme slug/path

Reply:
Good point, the full path of the theme would be good.